packages/uniforms-material/src/BoolField.tsx

```diff
@@ -33,11 +34,18 @@ function Bool(props: BoolFieldProps) {
     transform,
     value,
   } = props;
+  const theme = useTheme();
+  const formControlThemeProps = theme.props?.MuiFormControl;
   const SelectionControl =
     appearance === 'checkbox' || appearance === undefined ? Checkbox : Switch;
 
   return wrapField(
-    { fullWidth: true, margin: 'dense', ...props, component: 'fieldset' },
+    {
+      fullWidth: formControlThemeProps?.fullWidth ?? true,
+      margin: formControlThemeProps?.margin ?? 'dense',
```

Comment:
How about _not_ setting them at all if they are configured in the theme? That's probably even more idomatic.